tests/rgbpp/xudt/issuexudt/1-issue-xudt.ts

```diff
@@ -0,0 +1,119 @@
+import { addressToScript, getTransactionSize, scriptToHash } from '@nervosnetwork/ckb-sdk-utils';
```

Comment:
I suggest moving the `1-issue-xudt.ts` to the `xudt` directory and removing `issuexudt` directory

Reply:
Currently, L2 is used to distribute xudt for quick testing. The normal scenario should be to use L1 to distribute xudt. I consider keeping it for now and modify it later.

Reply:
> I suggest to move the `1-issue-xudt.ts` to the `xudt` directory and remove `issuexudt` directory

Currently, L2 is used to distribute xudt for quick testing. The normal scenario should be to use L1 to distribute xudt. I consider keeping it for now and modify it later.

Reply:
I see. I think renaming the `issuexudt` to `xudt-on-ckb` is better.

Reply:
https://github.com/ckb-cell/rgbpp-sdk/pull/169/commits/e6683d413f239fdc7c7c80b722cd0d8ed2d4aa34